openadapt/app/tray.py

```diff
@@ -0,0 +1,46 @@
+import os
+import threading
+from PIL import Image
+from pystray import Icon, Menu, MenuItem
+from openadapt.app.cards import quick_record, stop_record
+from openadapt.app.main import start, FPATH
+from openadapt import visualize
+
+app_thread = None
+g_tray = None
+
+
+def new_tray():
+    global g_tray
+
+    def on_exit():
+        os._exit(0)
```

Comment:
From https://stackoverflow.com/a/9591402/95989 is:

> os._exit calls the C function _exit() which does an immediate program termination. Note the statement "can never return".
> sys.exit() is identical to raise SystemExit(). It raises a Python exception which may be caught by the caller.

Can you please add a comment clarifying why we want to use os._exit(0) here instead of sys.exit()?

Reply:
the exception that sys.exit() raises doesn't actually stop the mainloop, and I'm not entirely sure why. For now I'm using os._exit(), but if I can incorporate a solution with signals + stop flag smoothly then I will switch.